.github/workflows/build.yml

```
@@ -11,51 +11,46 @@ jobs:
     name: Analyze with SonarCloud
     runs-on: ubuntu-latest
     steps:
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)